lib/jest-helpers.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const nunjucks = require('nunjucks')
+const cheerio = require('cheerio')
+
+const COMPONENT_DIR = 'src/components'
+
+nunjucks.configure(COMPONENT_DIR, {
+  trimBlocks: true,
+  lstripBlocks: true
+})
+
+function render (name, params) {
+  const output = nunjucks.render(name + '/template.njk', { params })
+  const $ = cheerio.load(output)
+  return { output, $ }
+}
+
+const yaml = require('js-yaml')
+const path = require('path')
+const fs = require('fs')
+
+function getExamples (name) {
+  const file = fs.readFileSync(
```

Comment:
What happens if you try to get examples for a non-existent component? Is the error message clear? If not, can we add error handling to this function.

Reply:
``` FAIL  src/components/back-link/template.test.js
  ● Test suite failed to run

    ENOENT: no such file or directory, open 'src/components/back-links/back-links.yaml'
```

Seems pretty clear to me, let me know what you think.